jmbitcoin/test/test_ecdh.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import * # noqa: F401
```

Comment:
You should drop all these imports.

Reply:
Thanks. I forgot.

Reply:
Should be resolved in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/f9e539535aaba572162474a640aacc6c87cc90da